search/.eslintrc.yml

```diff
@@ -6,6 +6,10 @@ extends:
   - eslint:recommended
 parserOptions:
   ecmaVersion: 12
+  sourceType: module
 rules:
   no-console: off
   radix: off
+  semi: 2
```

Comment:
Is this a valid option for semi? Looks like this should be "always" or "never":
https://eslint.org/docs/rules/semi

Reply:
The numbers are interchangeable with the text values, but I switched it to "error", which is the correct value.